packages/react/src/components/ImageCard/ImageHotspots.jsx

```diff
@@ -29,7 +29,23 @@ const propTypes = {
   /** optional features to enable or disable */
   hideZoomControls: PropTypes.bool,
   hideHotspots: PropTypes.bool,
-  hideMinimap: PropTypes.bool,
+  /* deprecated in favor of minimapBehavior below */
+  // eslint-disable-next-line consistent-return
+  hideMinimap: (props, propName, componentName) => {
+    if (__DEV__) {
+      const value = props[propName];
+      /* istanbul ignore else */
+      if (typeof value !== 'undefined') {
+        return new Error(
+          `${componentName}: '${propName}' prop is deprecated in favor of 'minimapBehavior="${
+            value ? 'hide' : 'show'
+          }"'.`
+        );
+      }
+    }
+  },
+  /* 'hide' always hides the minimap, 'show' always shows the minimap, 'onPan' only shows the minimap when panning an image */
+  minimapBehavior: PropTypes.oneOf(['hide', 'show', 'onPan']),
```

Comment:
onPan is a bit ambiguous, do we show or hide on pan? Can we call is showOnPan instead?

Reply:
yep. that's more clear.